WPMedia/fusion-news-theme-blocks
# Flex fix for left aligned logo

## Description
Prevent left aligned logo from overlapping hamburger menu (especially when a centered nav is placed in the center of the header (eg https://www.wsbtv.com/).

## Jira Ticket
[- [TMEDIA-](https://arcpublishing.atlassian.net/browse/TMEDIA-)](https://arcpublishing.atlassian.net/browse/TMEDIA-610)

## Acceptance Criteria
CMG have reported an issue with their logo overlapping the Sections Menu button.

It looks to be a combination of CMG's set up that's not full accounted for in theme blocks, they have some custom widgets in the right hand side of the navigation that is requiring set widths and as space becomes a premium it's being taking from areas that it shouldn't (which we can fix).

We need to ensure block elements maintain their space, this can be done by applying flex-shrink: 0 CSS property to the .news-theme-navigation-container .nav-left selector

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout tmedia-614-header-nav-chain`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles,@wpmedia/header-nav-block`
3. Update homepage so that the logo is left aligned:
![image](https://user-images.githubusercontent.com/833461/152833829-17effb51-6122-48c4-a27d-7a2e045f7124.png)
4. View published homepage.  Not much to test on our demo sites, but ensure proper responsiveness occurs from desktop to mobile


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [ x] Confirmed all the test steps a reviewer will follow above are working.
- [x ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x ] Confirmed this PR has unit test files
  - [x ] Ran `npm run test`, made sure all tests are passing

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
